doc/manual/src/language/constructs.md

```diff
@@ -217,18 +217,20 @@ three kinds of patterns:
     
     > **Warning**
     > 
-    > The `args@` expression is bound to the argument passed to the
-    > function which means that attributes with defaults that aren't
-    > explicitly specified in the function call won't cause an
-    > evaluation error, but won't exist in `args`.
+    > The `args@` expression is bound to the argument that is passed to the
+    > function. This does not take default values into account.
+    >
+    > Attributes with default values that aren't
+    > specified in the function call won't cause an
+    > evaluation error. But they also won't exist in `args`.
     > 
```

Comment:
```suggestion
```
I think this "won't cause an evaluation error stuff is a bad red herring. Who said it would?!

Reply:
That is the behavior of destructured args without defaults.